integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogRestHiveBackendIT.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.spark.iceberg;
+
+import com.datastrato.gravitino.auxiliary.AuxiliaryServiceManager;
+import com.datastrato.gravitino.spark.connector.iceberg.IcebergPropertiesConstants;
+import com.google.common.collect.Maps;
+import java.util.Map;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestInstance;
+
+/** This class use Iceberg RESTCatalog for test, and the real backend catalog is HiveCatalog. */
+@Tag("gravitino-docker-it")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SparkIcebergCatalogRestHiveBackendIT extends SparkIcebergCatalogIT {
+
+  private static final String icebergRestServiceName = "iceberg-rest";
+
+  @Override
+  protected Map<String, String> getCatalogConfigs() {
+    Map<String, String> catalogProperties = Maps.newHashMap();
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_BACKEND,
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_BACKEND_REST);
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_URI, icebergRestServiceUri);
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_WAREHOUSE, warehouse);
+
+    catalogProperties.put(
```

Comment:
the following configuration is for GravitinoServer, I don't think it take effects.

Reply:
https://github.com/datastrato/gravitino/pull/3194/files#diff-aab46d64a574ac80061556551b96d1bbe4ccc7a4a683e36e807890177aafc165R61-R66

i injected the service configs to IcebergRestService before starting server.

Reply:
> the following configuration is for GravitinoServer, I don't think it take effects.

If it does not take effect, there will be throw `table not found` exception.

Reply:
it's hacky here

Reply:
removed icebergRestService related configs from this class.